rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,13 +16,15 @@ toc::[]
 [id="rosa-q3-2024_{context}"]
 === Q3 2024
 
+* **{hcp-title} multi-architecture cluster update.** {hcp-title-first} clusters created before 25 July, 2024 will migrate to a multi-architecture image on their next upgrade allowing you to use {AWS} Arm-based Graviton instance types for your workloads. For more information, see xref:../upgrading/rosa-hcp-upgrading.adoc#rosa-upgrade-options_rosa-hcp-upgrading[Upgrading ROSA with HCP clusters].
+
 * **{hcp-title} cluster node limit update.** {hcp-title} clusters can now scale to 250 worker nodes. This is an increase from the previous limit of 180 nodes. For more information, see xref:../rosa_planning/rosa-hcp-limits-scalability.adoc#tested-cluster-maximums-hcp-sd_rosa-hcp-limits-scalability[ROSA with HCP limits and scalability].
 
 * **IMDSv2 support in {hcp-title}.** You can now enforce the use of the IMDSv2 endpoint for default machine pool worker nodes on new {hcp-title} clusters and for new machine pools on existing clusters. For more information, see xref:../rosa_hcp/terraform/rosa-hcp-creating-a-cluster-quickly-terraform.adoc#rosa-hcp-creating-a-cluster-quickly-terraform[Creating a default ROSA cluster using Terraform].
 
 * **Upgrade multiple nodes simultaneously.** You can now configure a machine pool to upgrade multiple nodes simultaneously. Two new machine pool parameters, `max-surge` and `max-unavailable`, give you greater control over how machine pool upgrades occur. For more information, see xref:../upgrading/rosa-hcp-upgrading.adoc#rosa-hcp-upgrading[Upgrading ROSA with HCP clusters].
 
-* **{hcp-title} Graviton (ARM) instance types.** You can now use {AWS} Arm-based Graviton instance types for your workloads in {hcp-title-first} clusters created after 24 July, 2024, see xref:../rosa_architecture/rosa_policy_service_definition/rosa-hcp-instance-types.adoc#rosa-sdpolicy-aws-instance-types-graviton_rosa-hcp-instance-types[AWS Graviton (ARM) instance types].
+* **{hcp-title} Graviton Arm-based instance types.** You can now use {AWS} Arm-based Graviton instance types for your workloads in {hcp-title-first} clusters created after 24 July, 2024, see xref:../rosa_architecture/rosa_policy_service_definition/rosa-hcp-instance-types.adoc#rosa-sdpolicy-aws-instance-types-graviton_rosa-hcp-instance-types[AWS Graviton Arm-based instance types].
```

Comment:
I'm not sure if "Graviton Arm-based instance" is an approved product name, but there's any possiblity you can conform the Arm mention with the way we're supposed to specify architecture names, that would be best: https://redhat-documentation.github.io/supplementary-style-guide/#bit-64-arm

Reply:
This is directly from AWS where ROSA is hosted. That is why we write it this way:
https://aws.amazon.com/about-aws/whats-new/2024/06/aws-codebuild-arm-based-workloads-graviton3/